README.md

```diff
@@ -323,61 +353,41 @@ local ts_utils = require 'nvim-treesitter.ts_utils'
 
 Check [`:h nvim-treesitter-utils`](doc/nvim-treesitter.txt) for more information.
 
-# User Query Extensions
-
-Queries are what `nvim-treesitter` uses to extract informations from the syntax tree, and they are
-located in the `queries/{lang}/*` runtime directories (like the `queries` folder of this plugin).
-
-`nvim-treesitter` considers queries as any runtime file (see `:h rtp`), that is :
-
-- if the file is in any `after/queries/` folder, then it will be used to extend the already defined
-  queries.
-- Otherwise, it will be used as a base to define the query, the first query found (with the highest
-  priority) will be the only one to be used.
-
-This hybrid approach is the most standard way, and according to that, here is some ideas on how to
-use is :
-
-- If you want to rewrite (or write) a query, don't use `after/queries`.
-- If you want to override a part of a query (only one match for example), use the `after/queries`
-  directory.
-
 # Troubleshooting
 
-Before doing anything make sure you have the latest version of this plugin and run `:checkhealth nvim_treesitter`.
-This will help you find where the bug might come from.
+Before doing anything, make sure you have the latest version of this plugin and run `:checkhealth nvim_treesitter`. It can also help to update the parsers via `:TSUpdate`.
 
-## Feature `X` does not work for `{language}`...
+#### Feature `X` does not work for `{language}`...
```

Comment:
Looks like these should be h2 `##`

Reply:
same as above; it's less of a heading and more of a structured list...

Reply:
(should probably be consistent with the module list, though)